pandas/core/indexes/datetimes.py

```diff
@@ -1719,7 +1721,7 @@ def freq(self, value):
     weekday_name = _field_accessor(
         'weekday_name',
```

Comment:
you can change this here to be day_name

Reply:
We discussed just depreciating `weekday_name` for now (and removing it in a future version)

Reply:
exactly, but you can actually remove the implementation. The deprecation still remains, but you can remove internal code.